examples/dts/dts_activity_sequence.py

```diff
@@ -52,18 +51,18 @@ def sequence(ctx: task.OrchestrationContext, _):
 arm_scope = "https://durabletask.io/.default"
 token_manager = AccessTokenManager(scope = arm_scope)
 
-metaData: list[tuple[str, str]] = [
+meta_data: list[tuple[str, str]] = [
     ("taskhub", taskhub_name)
 ]
 
 # configure and start the worker
-with DurableTaskSchedulerWorker(host_address=endpoint, metadata=metaData, secure_channel=True, access_token_manager=token_manager) as w:
+with DurableTaskSchedulerWorker(host_address=endpoint, metadata=meta_data, secure_channel=True, access_token_manager=token_manager) as w:
     w.add_orchestrator(sequence)
     w.add_activity(hello)
     w.start()
 
     # Construct the client and run the orchestrations
-    c = client.TaskHubGrpcClient(host_address=endpoint, metadata=metaData, secure_channel=True)
+    c = DurableTaskSchedulerClient(host_address=endpoint, metadata=meta_data, secure_channel=True, access_token_manager=token_manager)
```

Comment:
Similar feedback about token manager. Developers shouldn't have to worry about creating or configuring something like this because it's a lower-level detail unrelated to their business logic. Instead, the DurableTaskSchedulerClient and worker types should create an access token manager _internally_ and not make it a responsibility of the developer.